src/proxy/momento/src/metrics/util.rs

```diff
@@ -0,0 +1,25 @@
+use goodmetrics::{GaugeDimensions, GaugeFactory, HistogramHandle, SumHandle};
+
+pub fn proxy_sum_gauge(g: &GaugeFactory, name: &'static str) -> SumHandle {
+    g.dimensioned_gauge_sum("momento_proxy", name, Default::default())
+}
+
+pub fn proxy_request_ok_gauge(g: &GaugeFactory, rpc: &'static str) -> SumHandle {
+    g.dimensioned_gauge_sum("momento_proxy", "Ok", GaugeDimensions::new([("rpc", rpc)]))
```

Comment:
What is an “Ok”? I think this should be something more like “momento_request”, with “result” and “rpc” dimensions.

Reply:
Changed in [5fb0e22](https://github.com/pelikan-io/pelikan/pull/149/commits/5fb0e227ba5c9d96eee9e98b06ee1e8d6e555c29)